app/actions/constants.ts

```diff
@@ -83,3 +83,5 @@ export const LEDGER_GET_ADDRESSES_LOADED = "LEDGER_GET_ADDRESSES_LOADED";
 export const LEDGER_NEXT_PAGE = "LEDGER_NEXT_PAGE";
 export const LEDGER_PREV_PAGE = "LEDGER_PREV_PAGE";
 export const LEDGER_DERIVATION_PATH_CHANGE = "LEDGER_DERIVATION_PATH_CHANGE";
+
+export const ERROR_SET = "ERROR_SET";
```

Comment:
please rename it and all actions/reducers related to `CRITICAL_ERROR_*` to make it clear that this will be set only when cirtical error occurs and whole application should go into errored state.